events/app_watcher.go

```diff
@@ -16,22 +16,61 @@ type AppWatcher struct {
 }
 
 type InstanceMetrics struct {
-	Cpu prometheus.Gauge
+	Cpu               prometheus.Gauge
+	DiskBytes         prometheus.Gauge
+	DiskUtilization   prometheus.Gauge
+	MemoryBytes       prometheus.Gauge
+	MemoryUtilization prometheus.Gauge
 }
 
 func NewInstanceMetrics(instanceIndex int, registerer prometheus.Registerer) InstanceMetrics {
+	constLabels := prometheus.Labels{
+		"instance": fmt.Sprintf("%d", instanceIndex),
```

Comment:
Design thought: maybe InstanceMetrics shouldn’t need to attach this label, maybe it should be given a wrapped Registerer which always adds it? (Not a blocker for the PR but I’d be interested in people’s thoughts)

Reply:
Seems sensible but decided not to do it for times sake